module/VuFind/src/VuFind/Search/Solr/Results.php

```diff
@@ -453,22 +455,22 @@ public function getPartialFieldFacets(
     }
 
     /**
-     * Returns data on pivot facets for the last search
+     * Returns data on pivot facets for the last search.
      *
-     * @return ArrayObject        Flare-formatted object
+     * @return array Flare-formatted object
      */
-    public function getPivotFacetList()
+    public function getPivotFacetList(): array
     {
         // Make sure we have processed the search before proceeding:
         if (null === $this->responseFacets) {
             $this->performAndProcessSearch();
         }
 
         // Start building the flare object:
-        $flare = new \stdClass();
-        $flare->name = 'flare';
-        $flare->total = $this->resultTotal;
-        $flare->children = $this->responsePivotFacets;
-        return $flare;
+        return [
```

Comment:
Maybe the comment above should say "array" instead of "object" at this point. I also wonder if this particular change should be in a separate PR since it's a bit beyond just "adding typehints." That being said, if it's easier to just leave it here, I won't complain too much. :-)